src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -319,25 +345,25 @@ class Accounts extends Component {
                                     <Button className={`btn list ${this.state.viewToggle ? "active" : ""}`}
                                         onClick={(event) => {
                                             this.setState(
-                                                {"viewToggle": true},
+                                                { "viewToggle": true },
                                                 () => {
                                                     sessionStorage.setItem("AccountsState", JSON.stringify(this.state));
                                                 }
                                             );
                                         }}>
-                                        <ListView className={`icon ${this.state.viewToggle ? "active" : ""}`}/>
-                                             List View
+                                        <ListView className={`icon ${this.state.viewToggle ? "active" : ""}`} />
+                                        List View
                                     </Button>
                                     <Button className={`btn card ${this.state.viewToggle ? "" : "active"}`}
-                                            onClick={(event) => {
-                                                this.setState(
-                                                    {"viewToggle": false},
-                                                    () => {
-                                                        sessionStorage.setItem("AccountsState", JSON.stringify(this.state));
-                                                    }
-                                                );
-                                            }} >
-                                        <CardView className={`icon ${this.state.viewToggle ? "" : "active"}`}/>
+                                        onClick={(event) => {
```

Comment:
ideally, create a separate function that handles this setState call (i.e. `handleCardClick()`) reason being is creating + calling a function here is memory inefficient. This will create new references to this function which isn't necessary.